word-search.py

```diff
@@ -229,8 +229,39 @@ def checkWord():
     # checks if user found a correct word
     return
 
-def printWordList():
-    # prints all possible words to be found at game end
+def printWordList(gridWordList, foundWords):
+    # Prints all possible words to be found at game end
+    # Reveals player's answers and possible answers
+    
+    from collections import defaultdict
+
+    # Group words by their lengths from gridWordList
+    length_groups = defaultdict(list)
+    for word in gridWordList:
+        if len(word) >= 3:  # Only consider words with length 3 or more
+            length_groups[len(word)].append(word)
```

Comment:
We already filter for word length in the load_word_library function. Please remove :)